src/include/planner/plannodes/insert_plan_node.h

```diff
@@ -79,6 +79,17 @@ class InsertPlanNode : public AbstractPlanNode {
       return *this;
     }
 
+    /**
+     * @param index_oids vector of index oids to insert into
+     * @return builder object
+     */
+    Builder &AddIndexOids(std::vector<catalog::index_oid_t> &&index_oids) {
```

Comment:
The entire vector's coming in as an r-value reference. You should be able to just do `index_oids_ = index_oids;` since vector should have a move-assignment operator.

Reply:
Does this not replace all oids that were already present in the index_oids_ vector?

Reply:
Yeah it would replace all of them. Do we expect to have to append to the list of index oids multiple times? I feel like this is a one-shot call by the binder and then the list is sort of immutable.

Reply:
OK that makes sense, I should probably change it to "SetIndexOids" then.